x-pack/plugins/security_solution/public/detections/components/alerts_table/timeline_actions/alert_context_menu.test.tsx

```diff
@@ -83,182 +92,229 @@ const markAsOpenButton = '[data-test-subj="open-alert-status"]';
 const markAsAcknowledgedButton = '[data-test-subj="acknowledged-alert-status"]';
 const markAsClosedButton = '[data-test-subj="close-alert-status"]';
 const addEndpointEventFilterButton = '[data-test-subj="add-event-filter-menu-item"]';
-
-describe('InvestigateInResolverAction', () => {
-  test('it render AddToCase context menu item if timelineId === TimelineId.detectionsPage', () => {
-    const wrapper = mount(<AlertContextMenu {...props} timelineId={TimelineId.detectionsPage} />, {
-      wrappingComponent: TestProviders,
+const openAlertDetailsPageButton = '[data-test-subj="open-alert-details-page-menu-item"]';
+
+describe('Alert table context menu', () => {
```

Comment:
This diff looks worse than it actually is. Just renamed the describe block, grouped the actions by category, and added tests for the alert details page